README.md

```diff
@@ -154,7 +162,7 @@ Example command:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **[Comming Soon]**
+- [ ] **[Navigation Menu]**
 
 
```

Comment:
- [ ] @simplegoose Great work on your readme file, but as previously requested, kindly give credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)